modules/BuildTools.Tasks/Run.cs

```diff
@@ -16,14 +16,25 @@ public class Run
 #else
 #error This must be built either for an SDK or for BuildTools
 #endif
-        : RunBase
```

Comment:
This change drops some essential API such as `Arguments` and `Command`. It may be easier to instead change `RunBase` to inherit from `ToolTask`. There will be some overlap, such as EnvironmentVariables, and in those cases, you can remove the RunBase version.